GLASGOW | MAY 25 | MIRABELLE MORAH | MODULE - ONBOARDING | WIREFRAME

mirabellemorah opened the pull request:
<!--

You must title your PR like this:

REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME

For example,

London | May-2025 | Carol Owen | Wireframe

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

I am bringing up my work to be first checked then merged. 

I first adjusted the text, then also added images gotten from pinterest to make the blog page look engaging. Using color theory, I also changed the colors to 2 major colors, purple and red. I have gone through the instructions to ensure this meets the requirements and also tested things out. 3 new HTML pages were also added to represent unique articles. I have also done the following below:

- [x] Use semantic HTML tags to structure the webpage
- [x] Create three articles, each including a title, summary, and a link
- [x] Check a webpage against a wireframe layout
- [x] Test web code using [Lighthouse](https://programming.codeyourfuture.io/guides/testing/lighthouse)
- [x] Use version control by committing often and pushing regularly to GitHub

## Acceptance Criteria

- [x] Semantic HTML tags are used to structure the webpage.
- [x] The page scores 100 for Accessibility in the Lighthouse audit.
- [x] The page header includes a title and description.
- [x] The articles section has three unique articles, each including a title, summary, and a link.
- [x] The page footer is fixed to the bottom of the viewport.
- [x] The webpage is styled using a linked .css file.
- [x] The webpage is properly committed and pushed to a branch on GitHub.

## Questions

I am not sure I have any questions for now.

illicitonion reviewed (commented):
This generally looks great! Two things to consider about your footer:
1. It's meant to be centred, according to the wireframe you're building out
2. It's quite hard to read the text if it happens to overlap some of the background text/images

Please try to fix that up, otherwise this is looking great!

mirabellemorah commented:
Hello 👋🏽 thank you so much for your kind words and review. Ah you're right it was meant to be centered according to the wireframe. I have adjusted those now.

On #no2 and the overlap, I would have liked to keep the initial footer overlapping blend mode in overlay because I did not want the footer text color to clash harshly with the body texts and images, but to show up 'softly' (lack of a better word) at the bottom.
 
![screen-rec](https://github.com/user-attachments/assets/5b8be151-d7ad-406f-8847-cdd9e07dd679)
(original)

![Screenshot 2025-05-19 at 20 30 04](https://github.com/user-attachments/assets/7f8fcc2c-256a-4c1d-810f-5bdccbfe254c)
(adjusted)

Nonetheless based on your review I have adjusted both to center and taken off the blend mode now. All fixed. ✅



> This generally looks great! Two things to consider about your footer:
> 
> 1. It's meant to be centred, according to the wireframe you're building out
> 2. It's quite hard to read the text if it happens to overlap some of the background text/images
> 
> Please try to fix that up, otherwise this is looking great!

mirabellemorah commented:
Thanks so much for the caution @illicitonion, I'll do that next time. :)
